runtime/src/util/batch_return.rs

```diff
@@ -25,6 +25,20 @@ impl BatchReturn {
         Self { success_count: n, fail_codes: Vec::new() }
     }
 
+    pub fn of(codes: &[ExitCode]) -> Self {
+        let mut success_count: u32 = 0;
+        let mut fail_codes = vec![];
+        for code in codes {
```

Comment:
nit: seems like you could just use a batch return gen and add_success, add_fail

Reply:
I initially tried to implement this as a const fn, but was thwarted by need to change (annotate const) things in fvm libs. Done now.